cppcache/src/TcrChunkedContext.hpp

```diff
@@ -106,11 +106,11 @@ class TcrChunkedContext {
                     TcrChunkedResult* result, uint8_t isLastChunkWithSecurity,
                     const CacheImpl* cacheImpl);
 
-  ~TcrChunkedContext() = default;
+  ~TcrChunkedContext();
 
   const uint8_t* getBytes() const;
 
-  size_t getLen() const;
+  int32_t getLen() const;
```

Comment:
Why the type change? The implementation is returning the size of a `std::vector` which is `size_t`.

Reply:
This was rebase damage. I'm still combing.